src/ampycloud/data.py

```diff
@@ -514,9 +662,65 @@ def find_slices(self) -> None:
             pass
 
         # Finally, let's metarize these slices !
-        self.metarize(which='slices', base_lvl_alt_perc=self.prms['BASE_LVL_ALT_PERC'],
-                      base_lvl_lookback_perc=self.prms['BASE_LVL_LOOKBACK_PERC'],
-                      lim0=self.prms['MAX_HITS_OKTA0'], lim8=self.prms['MAX_HOLES_OKTA8'])
+        self.metarize(
+            which='slices',
+        )
+
+    def _merge_close_groups(self) -> None:
```

Comment:
This function is not for the faint hearted. It is important to test it properly and ensure we treat possible expections and corner cases.
Also, since it is specific to groups, why don't we apply this to layers too?

Reply:
added comment